foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/dao/VaultPropertyDao.java

```diff
@@ -48,7 +48,7 @@ public boolean checkEmpty() {
 
```

Comment:
i: File moved from extensions-configuration-store-vault. That service was the implemented to set the application.properties propertyDaoClass value. Because we are now getting that value from an env var, we no longer need to have vault in a separate module to override the application.properties.